source/includes/write/bulk-write.rb

```diff
@@ -0,0 +1,71 @@
+require 'bundler/inline'
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = "<connection string>"
+
+Mongo::Client.new(uri) do |client|
+  # start-db-coll
+  database = client.use('sample_restaurants')
+  collection = database[:restaurants]
+  # end-db-coll
+
+  # start-bulk-insert-one
+  insert_one = { insert_one: { name: 'Steve Rogers Cafe', borough: 'Brooklyn' } }
+  # end-bulk-insert-one
+
+  # start-bulk-update-one
+  update_one = { update_one: 
+    { 
+      filter: { name: 'Mountain View' },
+      update: { '$set': { borough: 'Queens' } }
+    }
+  }
+  # end-bulk-update-one
+
+  # start-bulk-update-many
+  update_many = { update_one: 
```

Comment:
Should this be update_many?
```suggestion
  update_many = { update_many: 
```

Reply:
Yup, good catch.